src/mongocxx/collection.cpp

```diff
@@ -98,11 +98,15 @@ cursor collection::find(bsoncxx::document::view filter, const options::find& opt
     scoped_bson_t filter_bson;
     scoped_bson_t projection(options.projection());
 
-    if (options.modifiers()) {
-        filter_builder << "$query" << bsoncxx::types::b_document{filter}
-                       << builder::stream::concatenate{
-                              options.modifiers().value_or(document::view{})};
-
+    if (options.sort() || options.modifiers()) {
+        filter_builder << "$query" << bsoncxx::types::b_document{filter};
+        if (options.modifiers()) {
+            filter_builder << builder::stream::concatenate{
+                options.modifiers().value_or(document::view{})};
+        }
+        if (options.sort()) {
+            filter_builder << "$orderby" << bsoncxx::types::b_document{*options.sort()};
```

Comment:
Oh yeah, this is what I wanted in my comment on your most recent PR! I'll add this comment there.